src/NServiceBus.Core/Audit/AuditConfigReader.cs

```diff
@@ -0,0 +1,75 @@
+﻿namespace NServiceBus
+{
+    using System;
+    using System.Diagnostics;
+    using Config;
+    using Logging;
+    using NServiceBus.Settings;
+    using Utils;
+
+    class AuditConfigReader 
+    {   
+        public class Result
+        {
+            public string Address;
+            public TimeSpan? TimeToBeReceived;
+        }
+        public static bool GetConfiguredAuditQueue(ReadOnlySettings settings, out Result result)
+        {
+            if (settings.TryGet(out result))
```

Comment:
Assuming we agree that from now on new/changed APIs should prefer code settings over config file/registry settings when both are specified? We should state that explicitly in https://github.com/Particular/Vision/issues/22